src/core/handlers/GraphQLHandler.test.ts

```diff
@@ -736,7 +736,8 @@ describe('run', () => {
         userId: 'abc-123',
       },
     })
-    const result = await handler.run({ request })
+    const requestId = 'requestId'
```

Comment:
Let's use either the `uuid4()` we already have or `crypto.randomUUID()` here. It's a little touch but the more your tests resemble the way your system works, the better.